core/src/tpu.rs

```diff
@@ -199,15 +201,22 @@ impl Tpu {
         )
         .unwrap();
 
+        let (forward_stage_sender, forward_stage_receiver) = unbounded();
```

Comment:
Agree. Backpressure handling could just be dropping txs on the floor and incrementing a counter. This would at least make it obvious if forwarding stage can't keep up